backend/src/bundles/users/user.controller.ts

```diff
@@ -38,6 +41,18 @@ class UserController extends BaseController {
             // isProtected: true, we can add it later and it will require token
             handler: () => this.findAll(),
         });
+
+        this.addRoute({
+            path: UsersApiPath.CURRENT,
+            method: 'GET',
+            isProtected: true,
+            handler: (options) =>
+                this.getCurrentUser(
+                    options as ApiHandlerOptions<{
+                        user: UserGetCurrentRequestDto;
```

Comment:
here should be UserAuthResponseDto, also change type in declaration file.
add swagger